src/coreclr/jit/CMakeLists.txt

```
@@ -362,6 +366,15 @@ convert_to_absolute_path(JIT_ARM_SOURCES ${JIT_ARM_SOURCES})
 convert_to_absolute_path(JIT_I386_SOURCES ${JIT_I386_SOURCES})
 convert_to_absolute_path(JIT_ARM64_SOURCES ${JIT_ARM64_SOURCES})
 
+# Include natvis file for Windows
+if (CLR_CMAKE_TARGET_WIN32)
+  set(JIT_NATVIS_SOURCE
+    clrjit.natvis
+  )
+  convert_to_absolute_path(JIT_NATVIS_SOURCE ${JIT_NATVIS_SOURCE})
+  add_linker_flag("/NATVIS:${JIT_NATVIS_SOURCE}")
+endif(CLR_CMAKE_TARGET_WIN32)
```

These should be host instead of target checks so that we add it for JITs targeting non-Windows that run on Windows.

```suggestion
if (CLR_CMAKE_HOST_WIN32)
  set(JIT_NATVIS_SOURCE
    clrjit.natvis
  )
  convert_to_absolute_path(JIT_NATVIS_SOURCE ${JIT_NATVIS_SOURCE})
  add_linker_flag("/NATVIS:${JIT_NATVIS_SOURCE}")
endif(CLR_CMAKE_HOST_WIN32)
```